suclass/cms/templates/su-header.html

```diff
@@ -0,0 +1 @@
+../../../lagunita/cms/templates/su-header.html
```

Comment:
However we do these symlinks, we should try to do so consistently.
We could either link to SUclass'  LMS or if we're going to link to Lagunita,
we could link directly to Lagunita's LMS (since that's the "real" file on disk).
In fact, we may want to do the latter because some operations can run into issues w/ too many levels of symlinks (symlink to a symlink to a symlink...).